### PR TITLE
[ET/AB][83212064] opening new tab when ctrl or meta (command) clicking

### PR DESCRIPTION
**Pull request by Ebtoulson**
- [x] jasmine tests pass
- [x] osx tested
- [x] windows tested

so `_openNewWindow` actually opens a new tab by triggering a `_blank` but to truly open a new window someone should be able to do a `shift` + click but this event isn't checked for.

This brings me to my next question...is ctrl + clicking something that people can override in their browser? If so are we fine with ignoring it?

**compromise:** Since ctrl + click opens a new tab in windows and command + click opens a new tab in OSX I decided to just check for both instead of checking the OS.

### CODE REVIEW

**Comment by Ebtoulson** [comment]
@c0 so i just noticed ur comment on this [issue](https://github.com/primedia/jquery-autotagging/issues/21). Do you think we should try the second approach?

**Comment by c0** [comment]
@Ebtoulson I have a [story](https://www.pivotaltracker.com/story/show/81968454) to remove `stopPropagation`. It has the potential to be bigger than it appears.

I think this PR is great for a patch release. Removing `stopPropagation` can be addressed in a minor version. Thoughts?

**Comment by Ebtoulson** [comment]
@c0 sounds good to me. I think removing the `stopPropagation` would definitely be a better solution in the long run since it would accommodate any custom user preferences / events.

Can I get a +1?

**Comment by c0** [comment]
Cool. I'll test it in AG.

File: spec/javascripts/ClickHandlerSpec.js [hunk 1]
```diff
@@ -140,6 +140,38 @@ describe('Click handler', function() {
...
+          customEvent = $.Event('click', { ctrlKey: true })
+          $(document).find('img.photo').trigger(customEvent);
+          waitsFor(function() {
+            return clickHandler._openNewWindow.wasCalled;
+          });
+          runs(function() {
+            expect(clickHandler._setDocumentLocation).not.toHaveBeenCalled();
+          });
+        });
+      });
```
Comment (mpelzsherman) [inline spec/javascripts/ClickHandlerSpec.js:194]: I really appreciate that you wrote these tests!

Should we also test that the WH request was fired?

File: src/click_handler.coffee [hunk 2]
```diff
@@ -66,7 +66,7 @@ define ['jquery'], ($) ->
       target = getClosestAttr('target')
       if @_followHrefConfigured(e, options, @wh) && @_shouldRedirect(href)
         e.preventDefault()
-        if target == "_blank"
+        if (target == "_blank") || e.ctrlKey || e.metaKey
```
Comment (mpelzsherman) [inline src/click_handler.coffee:69]: :+1: on this approach

File: spec/javascripts/ClickHandlerSpec.js (hunk 1 above)
Comment (Ebtoulson) [inline spec/javascripts/ClickHandlerSpec.js:194]: definitely wouldnt hurt

**Comment by c0** [comment]
:+1: Works great in AG.

**Pull request merged**